internal/commands/build.go

```diff
@@ -328,6 +327,11 @@ func validateBuildFlags(flags *BuildFlags, cfg config.Config, inputImageRef clie
 		return client.NewExperimentError("Exporting to OCI layout is currently experimental.")
 	}
 
+	if !inputImageRef.Layout() && flags.AppPath == "" {
```

Comment:
I fear that adding this warning to all cases under this condition would be too noisy. `pack build my-app` is a very common (and recommended) invocation. I think we should only warn when `my-app` happens to be a local path. 

This should do the trick:

```
if _, err := os.Stat(inputImageRef.Name()); err == nil {
    // image name exists as a local path, warn
}
```

We probably don't need to check `inputImageRef.Layout()` because those refs will always start with `oci://` I'm pretty sure

Reply:
Thank you for your suggestion. I like your idea of avoiding warnings every time users run pack build.👍